src/components/ui/AnimatedFadingIcon.js

```diff
@@ -1,10 +1,9 @@
 import React, { useRef, useState, useEffect } from "react";
 import { Animated, Easing } from "react-native";
-import { Icon } from 'native-base';
 
 // min opacity
 // duration is in miliseconds
-export const AnimatedFadeingIcon = ({name}) => {
+export const AnimatedFadingIcon = ({Icon}) => {
```

Comment:
I would keep using lowercase convention for consistency.
Maybe something like this:

```suggestion
export const AnimatedFadingIcon = ({icon: Icon}) => {...}
```